src/nomad_simulations/schema_packages/workflow/base_workflows.py

```diff
@@ -0,0 +1,141 @@
+from functools import wraps
+from typing import TYPE_CHECKING, Optional
+
+if TYPE_CHECKING:
+    from nomad.datamodel.datamodel import EntryArchive
+    from structlog.stdlib import BoundLogger
+
+from nomad.datamodel.data import ArchiveSection
+from nomad.datamodel.metainfo.workflow import TaskReference, Workflow
+from nomad.metainfo import SubSection
+
+from nomad_simulations.schema_packages.model_method import BaseModelMethod
+from nomad_simulations.schema_packages.outputs import Outputs
+
+
+def check_n_tasks(n_tasks: Optional[int] = None):
```

Comment:
not sure in which case if not defining n_tasks helpful?

Reply:
Yeah, I don't remember... let me double check

Reply:
yes, I took it out. Now I set the default to 1.